src/core/binding.h

```diff
@@ -256,6 +256,9 @@ QUIC_STATUS
 QuicBindingInitialize(
```

Comment:
I think it might just be best to have this take the `UdpConfig` directly for most args now.